### ISSUE TITLE
Voice Control Not Working in Image Caption, Gallery Caption, and a Paragraph Component in a Column Block

### ISSUE THREAD

**Issue by ChrissiePollock** [wordpress-mobile/WordPress-iOS#19175]
### Expected behavior
Commands such as Move to Beginning and Move to End would work as they do in paragraphs

### Actual behavior

The cursor does not move.

### Steps to reproduce the behavior

Activate Voice Commands on iOS.
Open a post and create a paragraph block and an image block.
Dictate to Move to Beginning. 
Note it works in the paragraph block and not in a caption.

This was reported in #5468942-zen regarding an iPad.

I replicated using an iPhone 7. 

Here is a screen recording of my test:

https://user-images.githubusercontent.com/2545184/184144205-17b9e48e-0639-47d1-b4e6-056eb6a40339.mp4

##### Tested on [device], iOS [version], WPiOS [version]

Iphone 7, iOS version 15.6, WP version 20.3.0.2

**Comment by dcalhoun** [comment]
I have a [PR pending review](https://github.com/WordPress/gutenberg/pull/44850) addressing the Image and Gallery block caption issue. 

I have a [draft PR](https://github.com/WordPress/gutenberg/pull/44992) addressing the Columns block. Unfortunately, the required fix introduces severe performance issues, so it is unlikely to merge in the near future. 

Depending on the outcomes of the aforementioned PRs, I may close this ticket and open a new one for deeply nested blocks specifically.

**Comment by dcalhoun** [comment]
iOS Voice Control for Image and Gallery captions was fixed in https://github.com/WordPress/gutenberg/pull/44850, which should be included in the WPiOS 21.1 release. 

Unfortunately, the broken iOS Voice control experience for nested text inputs remains a problem. To better track that bug, I am closing this issue in favor of a more targeted issue: https://github.com/WordPress/gutenberg/issues/45415.
